dnf5.spec

```diff
@@ -254,6 +262,8 @@ It supports RPM packages, modulemd modules, and comps groups & environments.
 %dir %{_datadir}/dnf5
 %dir %{_datadir}/dnf5/aliases.d
 %config %{_datadir}/dnf5/aliases.d/compatibility.conf
+%config %{_unitdir}/dnf5-offline-transaction.service
+%config %{_unitdir}/dnf5-offline-transaction-cleanup.service
```

Comment:
This cannot be a config file, as it's in `/usr`. It also doesn't need to be a config file.

Reply:
Right, thanks for catching that.